graphql-library/src/resolvers.js

```diff
@@ -0,0 +1,19 @@
+export default {
+  ping: () => 'Living Trello of Takeshi: GraphQL API v0.0.1',
+  board: ({id}, {Trello}) => 
+    Trello.call(`/boards/${id}/lists`)
+      .then(filterOutNonRecurringLists)
+      .then(lists => Promise.all(lists.map(mapListToPromise(Trello))))
+      .then(lists => ({lists}))
+      .catch(console.error)
+}
+
+function filterOutNonRecurringLists(lists) {
+  return lists.filter(list => list.name.toLowerCase().includes('recur'))
+}
+
+function mapListToPromise(Trello) {
+  return list => Trello.call(`/lists/${list.id}/cards`)
+    .then(cards => ({...list, cards}))
```

Comment:
Maybe it's better to keep the items in `list` scoped separately from `cards`. More future proof to changes in what trello returns in a list object, and it makes it easier for us to add more fields in the new object. 
```suggestion
    .then(cards => ({meta: list, cards}))
```

Reply:
Agree

Reply:
Wait, no I don't agree. I think it could be better if we just declare what properties are being queried, maybe have a look at the actual schema.js and index.js and see how the query works first?
```
function idProps({id, name, desc}) {
  return {id, name, desc}
}
```
Can do something like this and use it to deconstruct `list` and then for cards, something like
`cards: cards.map(idProps)` should do the trick